CHANGES

```diff
@@ -15,6 +15,12 @@ Tactic language
 
 - Support for fix/cofix added in Ltac "match" and "lazymatch".
 
+Installation
```

Comment:
If @maximedenes decides to put this in the 8.8.0 milestone, then an intermediary section title "Changes from 8.8+beta1 to 8.8.0" needs to be created.